rep-0126.rst

```diff
@@ -7,21 +7,6 @@ Content-Type: text/x-rst
 Created: 10-Feb-2012
 Post-History: 20-Feb-2012
 
-Table of Contents
-=================
-
-#. Abstract_
-#. Background_
-#. Specification_
-#. Motivation_
-#. Rationale_
-#. Concerns_
-#. ReferenceImplementation_
-#. References_
-#. Copyright_
-
-.. _Abstract:
```

Comment:
As I understand where other files included their own `..contents::` directive this file manually created its table of contents which is why it and the anchors below are being removed here.